io.github.dummerle.rare.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id": "io.github.dummerle.rare",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/rare",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=/usr/bin",
+        "--device=all",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "modules": [
+        {
+            "name": "desktop-entry",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D io.github.dummerle.rare.desktop /app/share/applications/io.github.dummerle.rare.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "io.github.dummerle.rare.desktop"
```

Comment:
instead you should use:

https://github.com/Dummerle/Rare/raw/1.8.9/misc/rare.desktop